index.html

```diff
@@ -129,8 +192,13 @@ <h3>Element Attributes</h3>
   </style>
 
   <script>
+  const init = async() => {   
     const reader = document.querySelector('m-camera-barcode-scanner');
 
+    let cameraId = await getCameraId('6366'); 
+    await reader.play(null, cameraId);
+    await reader.startScanning(true);
```

Comment:
As mentioned in a separate review https://github.com/MeldCX/matrix-component-camera-barcode-scanner/pull/5#pullrequestreview-754614252, we can replace this by one line
```
await reader.startScanning(true, null, cameraId);
```